### DISCUSSION TITLE
Quantos buttons tem em seu source code?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#1250] [Discussão]
Oi pessoal, faz tempo que não posto algo no fórum.

Fiquei curioso para saber uma coisa só: **Quantas vezes um componente button aparece em seu source code e em quantos arquivos?**, adapta para a versão de button que vc usa, pois isso depende da library que vc usa né.

No projeto atual que trabalho por exemplo tem 270 ocorrências de `Button` espalhados em 49 arquivos apenas dentro da pasta `/src`

E vocês?

![](https://media.giphy.com/media/l3q2RJBdaqJY2SV3O/giphy.gif)

**Comment by thulioph** [comment]
Aqui tem algumas variações de `Button` como: `<RouterButton />`, `<Button />`, `<BackButton />` e algumas outras.. Mas no geral deu isso ai o resultado:

![screen shot 2018-10-25 at 14 25 22](https://user-images.githubusercontent.com/2343288/47518713-1c6b7880-d862-11e8-9993-7c7ea9a53f05.png)

![screen shot 2018-10-25 at 14 25 43](https://user-images.githubusercontent.com/2343288/47518714-1c6b7880-d862-11e8-8e50-56a24aad7b7d.png)

**Comment by felipefialho** [comment]
Aqui usamos `mat-button` e suas variações para criar os botões, sendo assim, fiz essa busca:

<img width="389" alt="screen shot 2018-10-25 at 2 49 04 pm" src="https://user-images.githubusercontent.com/3603793/47519952-322e6d00-d865-11e8-8851-68f205944d43.png">

**Comment by simaodeveloper** [comment]
@felquis o que vai fazer com essa informação? gostaria de saber o propósito :D

**Comment by leandrocunha** [comment]
![screenshot from 2018-10-25 14-52-55](https://user-images.githubusercontent.com/1820920/47520151-c1d41b80-d865-11e8-8f24-babbfb1c37c3.png)

até achei que fosse mais

**Comment by rodrigets** [comment]
![image](https://user-images.githubusercontent.com/23198657/47523279-0e235980-d86e-11e8-9a49-1b31817e6a60.png)

Aqui tem bastante rs.

**Comment by felquis** [comment]
https://github.com/frontendbr/forum/issues/1250#issuecomment-433145819 @simaodeveloper 

Não vou fazer nada além de ler a thread mesmo haha

**Comment by caioincau** [comment]
172 vezes em 71 arquivos.

**Comment by felquis** [comment]
É pouco button :D
